src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,33 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private double bladeWidth;
```

Comment:
Consider adding access modifiers to the `bladeWidth` field to encapsulate the data properly.